lib/preparser.ts

```diff
@@ -108,7 +101,7 @@ function checkDeletedCommand(argvSlice: string[]): void {
 		version: string,
 		verb = 'replaced',
 	) {
-		exitWithExpectedError(stripIndent`
+		throw new ExpectedError(stripIndent`
```

Comment:
I think something went wrong with using `ExpectedError` here or elsewhere in this file. It should not be printing a stack trace below:
```
$ ./bin/balena local push
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
ExpectedError: Note: the command "balena local push" was replaced in CLI version v11.0.0.
Please use "balena push" instead.
    at Array.replaced (/Users/paulo/balena/balena-io/balena-cli/build/preparser.js:51:15)
    at checkDeletedCommand (/Users/paulo/balena/balena-io/balena-cli/build/preparser.js:80:21)
    at Object.run (/Users/paulo/balena/balena-io/balena-cli/build/app.js:15:5)
```
Compare with:
```
$ balena version -a
balena-cli version "12.15.0"
Node.js version "12.18.3"

$ balena local push
Note: the command "balena local push" was replaced in CLI version v11.0.0.
Please use "balena push" instead.

Additional information may be available with the `--debug` flag.
For help, visit our support forums: https://forums.balena.io
For bug reports or feature requests, see: https://github.com/balena-io/balena-cli/issues/
```

Reply:
Right, I'll revert that usage.